src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/User/UserProfileImage.html

```diff
@@ -0,0 +1,9 @@
+<adh-show-image 
+    data-ng-if="config.profile_images_enabled"
+    data-path="{{assetPath}}" 
+    data-format="{{format || 'thumbnail' }}" 
+    data-css-class="{{ cssClass }} user-profile-image" 
+    data-alt="{{ 'TR__USER_PROFILE_IMAGE_OF' |translate:{userName: userName} }}"
+    data-fallback-url="/static/images/avatar_fallback.svg"
```

Comment:
http://caniuse.com/#feat=svg

I have found that we can use them safely with IE > 8 in other projects. Do you have other experiences? Should I add a fallback to a png?